compiler-plugin/src/main/java/io/ballerina/stdlib/persist/compiler/utils/Utils.java

```diff
@@ -240,4 +247,74 @@ public static String getDatastore(CodeActionContext ctx) throws BalException {
             throw new BalException("error while reading persist configurations. " + e.getMessage());
         }
     }
+
+    public static String readStringValueFromAnnotation
+            (io.ballerina.compiler.syntax.tree.NodeList<AnnotationNode> annotationNodes, String annotation,
+             String field) {
+        if (annotationNodes == null) {
+            return null;
```

Comment:
In Java, Optionals are used in cases where the result can be empty. Shall we adopt that approach here ?